README.md

```
@@ -0,0 +1,11 @@
+# PHP Dependency Injection Benchmark
+
+This repository benchmarks different dependency injection containers.
+
+## Test Results
+
+Results from the latest automated run of the Dockerized benchmarks are written below.
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)